templates/dashboard.nginx.conf

```diff
@@ -10,7 +10,7 @@ server {
     server_name _;
     ssl_certificate {{ dashboard_tls_certificate_filepath }};
     ssl_certificate_key {{ dashboard_tls_key_filepath }};
-    ssl_protocols TLSv1.1 TLSv1.2;
+    ssl_protocols TLSv1.2;
```

Comment:
:+1: do we have 1.3 available in this version of openssl/nginx?

Reply:
yes, adding TLS 1.3